ga4gh/static/swagger.json

```diff
@@ -0,0 +1,172 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "GA4GH Reference Server",
+    "version": "0.2.0",
+    "title": "GA4GH Reference Server",
+    "contact": {
+      "email": "someone@ucsc.edu"
+    },
+    "license": {
+      "name": "Apache 2.0",
+      "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+    }
+  },
+  "host": "localhost:8000",
+  "tags": [
+    {
+      "name": "variants",
+      "description": "Variant endpoints",
+      "externalDocs": {
+        "description": "Find out more",
+        "url": "documentation site"
+      }
+    }
+  ],
+  "schemes": [
+    "http"
+  ],
+  "paths": {
+    "/variants/{id}": {
+      "get": {
+        "tags": [
+          "variants"
+        ],
+        "description": "Returns variant for a given ID",
+        "summary": "Find variants by ID",
+        "operationId": "getVariant",
+        "produces": [
```

Comment:
The current documentation extraction from Avro is problematic due to the way it is implemented on top of the Avro parser.

As part of the ProtBuf conversion, L-Shift wrote a documentation extractor from the ProtBuf specification files.

For Swagger, I think it makes sense to also generate the swarger JSON file template. from the schema.  To solve the URL issue, it might make sense to just server up the JSON file as dynamic content and avoid having to create static content on startup.  Need to understand the issues with load-balanced servers.  Their might not even be write access to the static data.

@david4096 is the host field even required?  It seems like the client already knows the URL it's connecting to and this would only matter if the swagger static files was coming from a different host.